addons/website/views/snippets/s_numbers_showcase.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_showcase" name="Numbers Showcase">
+    <section class="s_numbers_showcase pt80 pb80 o_colored_level">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="8" style="gap: 24px;">
+                <div class="o_grid_item g-height-6 g-col-lg-5 col-lg-5 justify-content-start" style="--grid-item-padding-y: 32px; --grid-item-padding-x: 32px; grid-area: 1 / 1 / 9 / 7; z-index: 1;">
```

Comment:
> I find this padding not necessary, especially on mobile.

If we remove this padding, there will be no space at all on mobile, which is worst 🤔 We could use `offset-col` classes but there will still be alignment issues:

| mobile | tablet |
|--------|--------|
| ![Capture d’écran 2024-08-06 à 21 50 14](https://github.com/user-attachments/assets/d1a26906-25ad-409d-b712-22a9e265aec5) | ![Capture d’écran 2024-08-06 à 21 54 16](https://github.com/user-attachments/assets/ea61daf2-c882-4672-8be7-6773b7de9ea3) |

Did I miss something ? 🤔

Reply:
>  there will be no space at all on mobile

This will be fixed apparently... all the new masonry layouts will look the same way when merged 🥹
Anyway, we have to reduce this padding regardless, `32px` it's really to much considering that the block has no background.